vagrant_config.json

```diff
@@ -139,9 +139,20 @@
           "action": "add" // one of add, remove. Default: add.
```

Comment:
if the default config paths are found => load them, otherwise, ignore.
If the default config paths are overridden => load them; if not found => exit with error message.
This is the same behavior with the extension_paths.